lib/listener.js

```diff
@@ -347,6 +347,38 @@ internals.Listener.prototype._subscribe = function (path, socket, next) {
 };
 
 
+internals.Listener.eachSocket = function (callback, options) {
+
```

Comment:
Add `options = options || {};` here, then drop the test in the _eachSocket method.